modules/lead/product-jenkins/variables.tf

```diff
@@ -34,3 +34,6 @@ variable "pipelines" {
 
 variable "vault_namespace" {}
 variable "vault_root_token_secret" {}
+variable "jenkins_pipeline_source" {
+  default = "git"
```

Comment:
Saw `github` used earlier. Is `git` what we want to default to?

Reply:
yeah, that's what we will continue to use for ignite labs